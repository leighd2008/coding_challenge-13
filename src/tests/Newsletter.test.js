import { shallow } from 'enzyme';
import React from 'react';
import Newsletter from '../components/Newsletter/Newsletter';

it('expect to render Newsletter component', () => {
	expect(shallow(<Newsletter />)).toMatchSnapshot();
})