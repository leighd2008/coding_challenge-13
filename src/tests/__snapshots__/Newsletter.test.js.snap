// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render Newsletter component 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Newsletter />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <img
          alt=""
          src="bg-whitewave-desktop.svg"
          style={
            Object {
              "width": "100%",
            }
          }
        />,
        <div
          className="content"
        >
          <div
            className="ml-form-embed"
            data-account="1252504:i9w0t9r0s7"
            data-form="1154434:v3f5g7"
          />
        </div>,
        <img
          alt=""
          className="svg-flip"
          src="bg-whitewave-desktop.svg"
          style={
            Object {
              "width": "100%",
            }
          }
        />,
      ],
      "id": "newsletter",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "alt": "",
          "src": "bg-whitewave-desktop.svg",
          "style": Object {
            "width": "100%",
          },
        },
        "ref": null,
        "rendered": null,
        "type": "img",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <div
            className="ml-form-embed"
            data-account="1252504:i9w0t9r0s7"
            data-form="1154434:v3f5g7"
          />,
          "className": "content",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "className": "ml-form-embed",
            "data-account": "1252504:i9w0t9r0s7",
            "data-form": "1154434:v3f5g7",
          },
          "ref": null,
          "rendered": null,
          "type": "div",
        },
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "alt": "",
          "className": "svg-flip",
          "src": "bg-whitewave-desktop.svg",
          "style": Object {
            "width": "100%",
          },
        },
        "ref": null,
        "rendered": null,
        "type": "img",
      },
    ],
    "type": "section",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <img
            alt=""
            src="bg-whitewave-desktop.svg"
            style={
              Object {
                "width": "100%",
              }
            }
          />,
          <div
            className="content"
          >
            <div
              className="ml-form-embed"
              data-account="1252504:i9w0t9r0s7"
              data-form="1154434:v3f5g7"
            />
          </div>,
          <img
            alt=""
            className="svg-flip"
            src="bg-whitewave-desktop.svg"
            style={
              Object {
                "width": "100%",
              }
            }
          />,
        ],
        "id": "newsletter",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "alt": "",
            "src": "bg-whitewave-desktop.svg",
            "style": Object {
              "width": "100%",
            },
          },
          "ref": null,
          "rendered": null,
          "type": "img",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <div
              className="ml-form-embed"
              data-account="1252504:i9w0t9r0s7"
              data-form="1154434:v3f5g7"
            />,
            "className": "content",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "className": "ml-form-embed",
              "data-account": "1252504:i9w0t9r0s7",
              "data-form": "1154434:v3f5g7",
            },
            "ref": null,
            "rendered": null,
            "type": "div",
          },
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "alt": "",
            "className": "svg-flip",
            "src": "bg-whitewave-desktop.svg",
            "style": Object {
              "width": "100%",
            },
          },
          "ref": null,
          "rendered": null,
          "type": "img",
        },
      ],
      "type": "section",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
